integration/test/test_hint_time.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+#
+#  Copyright (c) 2015, 2016, 2017, 2018, 2019, 2020, Intel Corporation
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions
+#  are met:
+#
+#      * Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in
+#        the documentation and/or other materials provided with the
+#        distribution.
+#
+#      * Neither the name of Intel Corporation nor the names of its
+#        contributors may be used to endorse or promote products derived
+#        from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+"""HINT_TIME
+
+"""
+
+import sys
+import unittest
+import os
+import glob
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
+from integration.test import geopm_context
+import geopmpy.io
+import geopmpy.error
+
+from integration.test import util
+from integration.experiment import machine
+if util.do_launch():
+    # Note: this import may be moved outside of do_launch if needed to run
+    # commands on compute nodes such as geopm_test_launcher.geopmread
+    from integration.test import geopm_test_launcher
+    geopmpy.error.exc_clear()
+
+class AppConf(object):
+    """Class that is used by the test launcher in place of a
+    geopmpy.io.BenchConf when running the hint_time benchmark.
+
+    """
+    def write(self):
+        """Called by the test launcher prior to executing the test application
+        to write any files required by the application.
+
+        """
+        pass
+
+    def get_exec_path(self):
+        """Path to benchmark filled in by template automatically.
+
+        """
+        bin_name = 'test_hint_time'
+        # Look for in place build
+        script_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Note this logic is different than it is in the template (which assumes an in place build).  This logic should be added to a function in the util module so it can be easily shared with the other exiting app configs in the test directory (including the template).